test/unit/registration.js

```diff
@@ -112,9 +114,11 @@ describe('Returning a constructor', function () {
   describe('when an extends option is specified', function () {
     var Div;
     var div;
+    var tagName;
 
     beforeEach(function () {
-      Div = skate('my-element', {
+      tagName = helpers.uniqueTagName('my-element');
```

Comment:
It would be nice to use destructuring assignment here, i.e.,

```js
{'my-element': tagName} = helpers.uniqueTagName('my-element');
```
but it seems for destructuring assigment doesn't work for existing `var`s (unless I'm mistaken).